### PR TITLE
Show quick permission subline on newline

### LINKED ISSUE #43225: Custom permission subline displays as part of headline on master
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

The list of permissions include as part of custom permissions must start on a new line (be a subline)
Screenshot from 2024-01-31 00-17-25

### PR DESCRIPTION
**Pull request by nfebe**
| Before | After |
| --------- | ------ |
| Screenshot from 2024-01-31 00-17-25 | Screenshot from 2024-02-04 17-39-31 |

- Show quick permission details/explanations on subline
- Fix icon colors, should be white (or component set color) when highlighted.

* Resolves: https://github.com/nextcloud/server/issues/43225

### CODE REVIEW

File: apps/files_sharing/src/views/SharingDetailsTab.vue [hunk 1]
```diff
@@ -993,7 +993,7 @@ export default {
 				span:nth-child(1) {
 					align-items: center;
 					justify-content: center;
-					color: var(--color-primary-element);
+					//color: var(--color-primary-element);
```
Comment (susnux) [inline apps/files_sharing/src/views/SharingDetailsTab.vue:996]: maybe just drop?

Comment (Fenn-CS) [reply apps/files_sharing/src/views/SharingDetailsTab.vue:996]: Sorry for the oversight, I didn't do the last push.

**Review by emoral435 — APPROVED** [review]
IMO, to me, this change looks way cleaner than having the smaller text right next to the big text. I will cc desginers to get them on the same page, though. cc @nextcloud/designers

**Review by szaimen — APPROVED** [review]
Yes, looks much nicer from design perspective (didnt test and didnt review the code)

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
backport?

**Comment by Fenn-CS** [comment]
> backport?

 think at least for 28! 27 seems fine.

**Pull request merged**